### ISSUE TITLE
AArch64 toolchain support?

### ISSUE THREAD

**Issue by thomas-t1** [gavinpugh/vs-android#139]
I have been using vs-android successfully for quite some time, however I suddenly had a customer requirement to port our projects to AArch64. It appears there is no support in the latest vs-android build for these. Unfortunately I have very little time to port to AArch64, and I need to decide quickly if I have to drop vs-android and go for supported NDK tools/build system/IDE or modify vs-android to support AArch64. From the little I could gather from in the msbuild files, I suspect it is possible to add a toolchain by modifying the msbuild files?

1. Can the AArch64 toolchain(s) be added through msbuild file alone?
2. Do I need to modify the code as well?
3. Is there some kind of documentation for adding a new toolchain? Any hints would be appreciated.

**Comment by thomas-t1** [comment]
I added support for gcc 4.9 aarch64/arm64v8-a in the VS2012 part, is there any interest in this patch?

**Comment by phofman** [comment]
I am interested :)

**Comment by thomas-t1** [comment]
I don't really know how or if I would be allowed to contribute to the vs-android repo. the author does not seem to be very active, so I've just attached a zip (renamed to png) archive with the necessary changes, replace/add the files in the vs-android\MSBuild directory. Good luck.

![aarch64-linux-android-4 9_zip](https://cloud.githubusercontent.com/assets/12417334/7947883/3a25e112-0980-11e5-98cf-cdc173619864.jpg)

**Comment by phofman** [comment]
That's better than the image-zip trick

**Comment by thomas-t1** [comment]
Yeah, old dog, new tricks... I'm more of a TFS guy myself, but times change ;)
